### PR TITLE
fix: Handle already created event bus error

### PR DESCRIPTION
**Pull request by khanek**

### CODE REVIEW

File: services/backend/scripts/install_localstack_fixtures.sh [hunk 1]
```diff
@@ -14,9 +14,25 @@ function wait_for_s3 {
...
-
+function is_event_bus_exists {
+  out=$(
+    aws --region=eu-west-1 --endpoint-url="$LOCALSTACK_URL" events list-event-buses | jq ".EventBuses[] | select(.Name == \"$1\") | .Name"
+  )
+  if [[ -z "$out" ]]; then
+    return 1
+  else
+    return 0
+  fi
```
Comment (Toreno96) [inline services/backend/scripts/install_localstack_fixtures.sh:25]: You could do `return [[ -n "$out" ]]`, I think.

Comment (khanek) [reply services/backend/scripts/install_localstack_fixtures.sh:25]: `./scripts/install_localstack_fixtures.sh: line 21: return: [: numeric argument required` but I think we could replace with:
`test -n "$out"
  return $?`
What do you think @Toreno96 ?

Comment (Toreno96) [reply services/backend/scripts/install_localstack_fixtures.sh]: Let's stay w/ previous version then, it's more readable IMHO.

Comment (khanek) [reply services/backend/scripts/install_localstack_fixtures.sh:25]: done :)

**Review by Toreno96 — APPROVED** [review]

**Review by pziemkowski — APPROVED** [review]

**Pull request merged**